dfc.providerportal.findanapprenticeship/src/dfc.providerportal.findanapprenticeship.unittests/Integration/GetApprenticeshipsAsProviderIntegrationTests.cs

```diff
@@ -0,0 +1,115 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Net;
+using System.Net.Http;
+using System.Threading;
+using System.Threading.Tasks;
+using Dfc.Providerportal.FindAnApprenticeship.Functions;
+using Dfc.Providerportal.FindAnApprenticeship.Helper;
+using Dfc.Providerportal.FindAnApprenticeship.Interfaces.Helper;
+using Dfc.Providerportal.FindAnApprenticeship.Interfaces.Services;
+using Dfc.Providerportal.FindAnApprenticeship.Models;
+using Dfc.Providerportal.FindAnApprenticeship.Services;
+using Dfc.Providerportal.FindAnApprenticeship.Settings;
+using FluentAssertions;
+using LazyCache;
+using Microsoft.ApplicationInsights;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Documents.Client;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.Extensions.Options;
+using Moq;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Xunit;
+
+namespace Dfc.ProviderPortal.FindAnApprenticeship.UnitTests.Integration
+{
+    public class GetApprenticeshipsAsProviderIntegrationTests
+    {
+        private readonly TelemetryClient _telemetryClient;
+        private readonly IAppCache _appCache;
+        private readonly Mock<IConfiguration> _configuration;
+        private readonly Mock<ICosmosDbHelper> _cosmosDbHelper;
```

Comment:
I've been naming mock fields like this:
```suggestion
        private readonly Mock<ICosmosDbHelper> _mockCosmosDbHelper;
```
But now I think about it that might be bad hungarian. It does make the calls a tiny bit easier to read, I'd double take at `_cosmosDbHelper.Object` otherwise. I'd be interested in your thoughts on the matter.

Reply:
Ultimately the property name is well described so far as what it is and what it's doing either way. So I don't think I'm too fussed about this. Also if in the future someone decides to put a concrete object there, which would be valid in the context of this test, then the naming convention as it currently stands would not require a change.